networking/gen-coal.sh

```diff
@@ -44,15 +44,12 @@ function get_server
 	gc_server=$uuid
 }
 
-#
-# Coal always has e1000g0 for the external network.
-#
 function get_mac
 {
 	local mac
 
-	mac=$(dladm show-phys -mpo address e1000g0)
-	[[ $? -eq 0 ]] || fatal "failed to run dladm"
+	mac=$(nictagadm list | awk '/^external/{print $2}')
```

Comment:
Yes. If I'd noticed it existed I would have. Clearly I should update this. Thanks!